src/test/java/picard/sam/markduplicates/AbstractMarkDuplicatesCommandLineProgramTester.java

```diff
@@ -163,15 +164,20 @@ public void test() throws IOException {
                 for (final SAMRecord record : reader) {
                     outputRecords++;
                     final String key = samRecordToDuplicatesFlagsKey(record);
-
+                    if (!this.duplicateFlags.containsKey(key)) {
+                        System.err.println("DOES NOT CONTAIN KEY: " + key);
```

Comment:
If what's desired here is adding more information to the assertion, the way to do it is to add the assertion message to the assert
```
                    Assert.assertTrue(this.duplicateFlags.containsKey(key), "DOES NOT CONTAIN KEY: " + key);
```